src/spyglass/lfp/v1/lfp.py

```diff
@@ -162,28 +162,16 @@ def make(self, key):
                 "valid times",
             )
         )
-
-        tmp_valid_times = (
-            IntervalList
-            & {
+        interval_key = IntervalList().cautious_insert1(
+            key={
                 "nwb_file_name": key["nwb_file_name"],
                 "interval_list_name": key["interval_list_name"],
-            }
-        ).fetch("valid_times")
-        if len(tmp_valid_times) == 0:
-            IntervalList.insert1(
-                {
-                    "nwb_file_name": key["nwb_file_name"],
-                    "interval_list_name": key["interval_list_name"],
-                    "valid_times": lfp_valid_times,
-                    "pipeline": "lfp_v1",
-                },
-                replace=True,
-            )
-        elif not np.allclose(tmp_valid_times[0], lfp_valid_times):
```

Comment:
Similar concern about still needing to check for matching as in comment for LFPBand